UPGRADE-3.x.md

```diff
@@ -1,13 +1,23 @@
 UPGRADE 3.x
 ===========
 
-UPGRADE FROM 3.19 to 3.20
```

Comment:
This change looks strange, shouldnt be done on 3.x if any?

Reply:
It's coming from 3.x